forch/forchestrator.py

```diff
@@ -436,9 +445,43 @@ def _restore_states(self):
     def _restore_faucet_config(self, timestamp, config_hash):
         config_info, faucet_dps, _ = self._get_faucet_config()
         self._update_config_warning_varz()
-        assert config_hash == config_info['hashes'], 'config hash info does not match'
+
+        if config_hash == config_info['hashes']:
+            self._attempt_cancel_config_hash_clash_timer()
```

Comment:
I think in these cases I'd leave out the "attempt" -- it's not semantically meaningful

Reply:
Not using a seprate timer now.